Cargo.toml

```diff
@@ -645,28 +645,42 @@ move-vm-test-utils = { path = "third_party/move/move-vm/test-utils", features =
 ] }
 move-vm-types = { path = "third_party/move/move-vm/types" }
 
+### Cargo profiles
+# See here for reference https://doc.rust-lang.org/cargo/reference/profiles.html
+
+# Release build
+# Major difference here is LTO which give a significant performance gain
 [profile.release]
+# Include debuginfo
 debug = true
+# Panic on overflow
 overflow-checks = true
+# Perform thin lto, achieving most of the benefit with less compile time
+lto = "thin"
+# Do not split crates into multiple compiled units
+codegen-units = 1
 
-# The performance build is not currently recommended
-# for production deployments. It has not been widely tested.
+# Correctness build, used for forge / testing
+# This is ostenisbly the same as release minus lto
+[profile.correctness]
+inherits = "release"
+lto = false
+
+# Performance build
+# Previously the LTO build, duplicate until we remove references to this
 [profile.performance]
 inherits = "release"
-opt-level = 3
```

Comment:
missing this one in release profile?

Reply:
This is actually unnecessary because it is default in the release profile which we inherit from

https://doc.rust-lang.org/cargo/reference/profiles.html#release